index.js

```diff
@@ -14,8 +14,8 @@ async function run() {
     const taskDefinitionFile = core.getInput('task-definition', { required: false });
     const containerName = core.getInput('container-name', { required: true });
     const imageURI = core.getInput('image', { required: true });
-    const environmentVariables = core.getInput('environment-variables', { required: false });
-    const envFiles = core.getInput('env-files', { required: false });
+    const environmentVariables = core.getInput('environment-variables', { required: false });    
+    const envFiles = core.getInput('env-files', { required: false });    
```

Comment:
nit: Looks like unintentional extra lines are added at the end of these lines?

Reply:
Fixed the extra line. Resolving the comment.